Pythonwin/Scintilla/src/LexGen.py

```diff
@@ -4,7 +4,7 @@
 # Regenerate the Scintilla and SciTE source files that list
 # all the lexers and all the properties files.
 # Should be run whenever a new lexer is added or removed.
-# Requires Python 2.4 or later
+# Requires Python 3.0 or later
```

Comment:
scintilla is "vendored" but I don't think yet "forked" - so we should probably skip mods there.

Reply:
By "mods" you mean "modifications" right?
Make sense if it's vendored.

I'm not certain exactly what you mean by "forked" (I'm lacking a bit of context here). As in it will come to live in pywin32 from somewhere else? Or as in it will be taken out of pywin32 to exist on its own?

> Until https://github.com/mhammond/pywin32/pull/1986 lands it will be tricky to look closely, but a very quick look seems great.

I'm in no hurry, let's do things one step at a time with isolated changes :)

Reply:
Btw if part of this PR is blocked because it touches vendored code. I can simply split off those changes.

Reply:
```suggestion
# Requires Python 2.4 or later
```
I'll revert this for now. It's only a comment change anyway